### DISCUSSION TITLE
Can change color Column chart by value ?

### DISCUSSION THREAD

**Discussion by namnguyenhai** [ant-design/ant-design#42788] [Q&A] [answered]
Example I have data like this: 
```
const data = [
        {
            type: '1-3秒',
            value: 0.16,
        },
        {
            type: '4-10秒',
            value: 0.25,
        },
```
My config: 
```
const config = {
        data,
        xField: 'type',
        yField: 'value',
        color: ({ value}) => {
            if (value<0.2) {
                return '#F4664A';
            }

            return '#5B8FF9';
        },
```
If I use code above my Column chart color not change. But when I check Column char by type name it changes color. How can I change Column chart color by value ?

Thanks

**Comment by MadCcc** [comment]
In which component you used config mentioned? Have you opened this discussion in wrong place?

**Reply by namnguyenhai** [reply]
I use component Column from '@ant-design/plots'

**Reply by MadCcc** [reply, marked as answer]
You should open the issue here: https://github.com/ant-design/ant-design-charts

**Reply by namnguyenhai** [reply]
I have created issue at: https://github.com/ant-design/ant-design-charts/issues/1957
